sympy/functions/special/polynomials.py

```diff
@@ -52,7 +52,7 @@ def _eval_conjugate(self):
 
 
 class jacobi(OrthogonalPolynomial):
-    r"""
```

Comment:
I think you should leave the `r` here: it marks this as a "raw" docstring with backslashes needing to not be interpreted as the beginning of an escape sequence.